commands/host.vegman

```diff
@@ -196,20 +205,30 @@ function cmd_virtualmedia_status {
 # @restrict cmd_virtualmedia_mount admin
 # @doc cmd_virtualmedia_mount
 # Mount a virtual media
+#   -m MODE     - The mount mode to selects the appropriate way to inject an image.
+#      legacy   - accept images from an external server through a particular protocol.
+#      nbd      - exports images from external NBD server.
 #   --usb       - Mount as a USB interface type
 #   --usb-ro    - Mount as a USB interface type (read-only)
 #   --hdd       - Mount as an HDD interface type
 #   --cdrom     - Mount as a CD/DVD/BD-ROM interface type (read-only),
 #                 based on file size.
 #   FILE        - An image file URL. Supported protocols are:
-#                 smb, scp, nfs, ftp, sftp, http, https
+#                 smb, scp, nfs, ftp, sftp, http, https.
+#                 LEGACY mode required.
+#   ADDRESS     - An NBD external server address.
+#                 NBD mode required.
+#   -e EXPORT     An NBD export name.
```

Comment:
I have a doubt: Is it really required to specify? Can't it be predefined?

Reply:
The old-style NBD protocol doesn't require an export name bat the new style is.
The modern versions of NBD-servers provide new-style protocol and mark old-style as deprecated; some implementations don't support the old-style protocol. Additionally, to use an old-style protocol, there must have a separate port on each export.